app/src/main/java/com/pallaw/swipeandlearnf/feature/adapter/QuestionsAdapter.kt

```diff
@@ -0,0 +1,85 @@
+package com.pallaw.swipeandlearnf.feature.adapter
+
+import android.app.Activity
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.RelativeLayout
+import android.widget.TextView
+import androidx.core.view.isVisible
+import androidx.recyclerview.widget.RecyclerView
+import com.pallaw.swipeandlearnf.R
+import com.pallaw.swipeandlearnf.feature.data.CardQuestionData
+import com.yuyakaido.android.cardstackview.CardStackListener
+import com.yuyakaido.android.cardstackview.Direction
+
+class QuestionsAdapter(
```

Comment:
@akshat-sahijpal move this class to Ui/game folder

Reply:
done